profile/README.md

```diff
@@ -26,12 +28,23 @@ it's definitely not the only thing that makes it amazing... you are!
 
 Practice kindness, be welcoming, and always assume that we all have something to learn. Take a minute to read more about our [code of conduct](CODE_OF_CONDUCT.md).
 
-## Contributing
+## How we work
 
+### Contributing
 While each community in our collection of Octokit communities is unique and has its own individuality, we have standard approaches across all of our repos
 to lower barriers to entry and reduce the friction of getting things done. Make sure to have a look at each repository's `CONTRIBUTING` docs above when
-making contributions. For more information on how our team prioritizes and responds to issues, PRs, and discussions, please see [this document](community/prioritization_response.md).
-Oh, and thanks for being amazing and helping to both improve our communities and code! ❤️
+making contributions. 
+
+### Team and Community processes
+- For more information on how our **team prioritizes** and responds to issues, PRs, and discussions, please see [this document](community/prioritization_response.md).
```

Comment:
This link 404s: the relative link needs to be prefixed with a `../` since this README is inside the `profile` directory.